### PR TITLE
Fixes issue where todos were deleted from a different engine

### PR DESCRIPTION
**Pull request by scalvert**
Fixes an issue reported in https://github.com/scalvert/eslint-formatter-todo/issues/80.

While this wouldn't directly plague this lib due to the extra file-based scoping we do here, it ensures that we're still applying the same scope filtering, and ensures this with a test.

### CODE REVIEW

**Review by rwjblue — COMMENTED** [review]
LGTM once #1737 lands and this is rebased

**Review by rwjblue — APPROVED** [review]

File: test/acceptance/todo-cli-test.js
Comment (rwjblue) [inline test/acceptance/todo-cli-test.js:164]: `.only` 🥩 🔪

**Review by rwjblue — CHANGES_REQUESTED** [review]

File: test/acceptance/todo-cli-test.js
Comment (rwjblue) [reply test/acceptance/todo-cli-test.js:164]: hopefully this would have failed CI, if not can you file an issue?

Comment (scalvert) [reply test/acceptance/todo-cli-test.js]: It should have failed linting, as that rule is for sure enabled.

**Review by rwjblue — APPROVED** [review]

**Pull request merged**
